scripts/fast-reboot

```diff
@@ -247,10 +247,14 @@ function backup_database()
             end
         end
     " 0 > /dev/null
-    sonic-db-cli SAVE > /dev/null
-    #TODO : need a script to copy all rdb files if there is multiple db instances config
-    docker cp database:/var/lib/redis/$REDIS_FILE $WARM_DIR
-    docker exec -i database rm /var/lib/redis/$REDIS_FILE
+
+    # move all db data into instances where APPL_DB locates
+    migrate_warmboot_database APPL_DB
```

Comment:
>migrate_warmboot_database [](start = 4, length = 25)

The name is misleading. It is not directly related to warmboot.
Suggestion `centralize_database`

Reply:
updated